addons/event/static/src/client_action/event_registration_summary_dialog.xml

```diff
@@ -3,60 +3,77 @@
 <templates xml:space="preserve">
 
     <t t-name="event.EventRegistrationSummaryDialog" >
-       <Dialog size="'md'" title="'Registration Summary'">
+       <Dialog size="'md'" title="'Home'">
             <div class="row">
-                <div class="col-lg-8 offset-lg-2">
-                    <div class="text-center mb-3" t-if="registration.partner_id">
-                        <img t-attf-src="/web/image/res.partner/{{registration.partner_id}}/avatar_128" alt="Registration" class="o_image_64_cover"/>
-                    </div>
-                    <div t-if="registration.status === 'confirmed_registration'" class="alert alert-success text-center" role="alert">
-                        <t t-out="registration.name"/> is successfully registered
+                <div class="col-lg-10 w-100 fs-2">
+                    <div t-if="registration_status.value === 'confirmed_registration'" class="alert alert-success text-center" role="alert">
+                        <i class="fa fa-solid fa-check-circle"/><br/>
+                        <span>Successfully registered!</span>
                     </div>
                     <div t-else="" class="alert alert-warning text-center" role="alert">
-                        <t t-if="registration.status === 'need_manual_confirmation'">
+                        <i class="fa fa-solid fa-exclamation-circle"/><br/>
+                        <t t-if="registration_status.value === 'need_manual_confirmation'">
                             <span>This ticket is for another event<br/>
                             Confirm attendance for <t t-out="registration.name"/>?</span>
                         </t>
-                        <t t-elif="registration.status === 'not_ongoing_event'">
+                        <t t-elif="registration_status.value === 'not_ongoing_event'">
```

Comment:
Scanning an invalid ticket is not going to open the registration summary dialog, it will just close the scanner and display an error in a toaster 
![image](https://github.com/odoo/odoo/assets/30419015/806ef921-7cbc-4245-9815-2e39125eb5ab)